src/Tribe/Views/V2/View.php

```diff
@@ -1940,7 +1940,13 @@ protected function get_ical_data() {
 		 *
 		 * @param string $text The default link text, which is "Export Events".
 		 */
-		$link_text  = apply_filters( 'tribe_events_ical_export_text', __( 'Export Events', 'the-events-calendar' ) );
+		$link_text  = apply_filters(
```

Comment:
:warning: Equals sign not aligned correctly; expected 1 space but found 2 spaces

<sub>hash: 6c86fda4c7c160febf39a1eb0a4eeb5a</sub>